hack/update_codegen.sh

```diff
@@ -18,51 +18,56 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+TMP_VENDOR_DIR=gen-vendor
+
+function cleanup() {
+  echo "Cleaning up..."
+  rm -rf ${TMP_VENDOR_DIR}
+  rm sigs.k8s.io
+}
+
+trap cleanup EXIT
 GO_CMD=${1:-go}
 NFD_ROOT=$(realpath $(dirname ${BASH_SOURCE[0]})/..)
 
-go mod vendor
+"${GO_CMD}" mod vendor
```

Comment:
Remind me: we need the `vendor/` for running `go generate`? And thus can't just do `go mod vendor -o "$TMP_VENDOR"`?

Reply:
`"${GO_CMD}" generate ./cmd/... ./pkg/... ./source/...` fails if things are not in `vendor`